var/spack/repos/builtin/packages/py-protobuf/package.py

```diff
@@ -20,7 +20,16 @@ class PyProtobuf(PythonPackage):
     variant('cpp', default=False,
             description='Enable the cpp implementation')
 
+    version('3.20.1',  sha256='d3449b4df32ca155f8cd71051db916b2bf74cc925760e269c0acf09b48ec1cb8')
```

Comment:
These checksums are wrong...

Reply:
I obtained them with `spack checksum py-protobuf`, what is the correct command these days?

Reply:
Looks like `spack checksum` is broken (tagged you on the issue). For now, you'll have to manually download and checksum them.

Reply:
I updated the hashes with what PyPi publishes.